tests/integration-tests-topologies/src/main/java/org/apache/pulsar/tests/containers/S3Container.java

```diff
@@ -0,0 +1,55 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pulsar.tests.containers;
+
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * S3 simulation container
+ */
+@Slf4j
+public class S3Container extends ChaosContainer<S3Container> {
+
+    public static final String NAME = "s3";
+
+    private static final String IMAGE_NAME = "adobe/s3mock:latest";
```

Comment:
Please use apachepulsar/s3mock:latest

Reply:
Once the issue in adobe/s3mock is fixed, I will change it back from apachepulsar/s3mock.

Reply:
There's no harm in making it use apachepulsar/s3mock forever. It's probably better, since it won't change under our feet.